src/e2e/java/teammates/e2e/cases/lnp/FeedbackSessionSubmitLNPTest.java

```diff
@@ -252,6 +252,16 @@ protected ListedHashTree getLnpTestPlan() {
         return testPlan;
     }
 
+    @Override
+    protected double getErrorRateLimit() {
```

Comment:
I think a better approach is to create a `LNPSpecificiation` somewhat class to hold the limit and put into the super class of the LNPTest.

Reply:
Moved logic for specification and result verification to `LNPSpecification` class.